extensions/aria_extension_tosca/simple_v1_0/modeling/data_types.py

```diff
@@ -397,8 +397,8 @@ def coerce_value(context, presentation, the_type, entry_schema, constraints, val
 
     if hasattr(the_type, '_coerce_value'):
         # Delegate to '_coerce_value' (likely a DataType instance)
-        return the_type._coerce_value(context, presentation, entry_schema, constraints, value,
-                                      aspect)
+        return the_type._cast_value(context, presentation, entry_schema, constraints, value,
```

Comment:
I tried to understand this change and am stumped. I am pretty sure that it's simply a mistake.